### PR TITLE
change library name

### PR DESCRIPTION
**Pull request by dvarrel**

### CODE REVIEW

**Comment by per1234** [comment]
Hello @dvarrel. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

Note that it appears that, in addition to changing the library URL, which is now in progress, you are also attempting to change the library's name:

https://github.com/dvarrel/ESPping/commit/f33c784593b98667466da86268faf8acb76b9699#diff-f267f9ae2f5d5d56a1dbb305e95d40d4408d9ee7f71357f1f67fb18c29b7f082R1

```diff
- name=ESP_Ping
+ name=ESPping
```

If you want to do that, you must first make a release/tag that contains the change. The current release version of the library still uses the currently registered name "**ESP_Ping**":

https://github.com/dvarrel/ESPping/blob/1.0.0/library.properties#L1

```text
name=ESP_Ping
```

You can learn how to request a name change here:

https://github.com/arduino/library-registry/blob/main/FAQ.md#how-can-i-change-a-librarys-name

**Review by umbynos — APPROVED** [review]

**Comment by umbynos** [comment]
Done in https://github.com/arduino/library-registry/pull/2256#issuecomment-1362665766

**Pull request merged**
